plugins/inputs/aliyuncms/aliyuncms.go

```diff
@@ -237,17 +272,34 @@ func (s *AliyunCMS) Init() error {
 
 	s.measurement = formatMeasurement(s.Project)
 
+	//Check regions
+	if len(s.DiscoveryRegions) != 0 {
+		s.Log.Warn("'discovery_regions' is deprecated. Use 'regions' instead.")
+	}
+
+	if len(s.Regions) == 0 {
+		s.Regions = s.DiscoveryRegions
+	} else {
+		s.Log.Warn("'discovery_regions' is deprecated and ignored in favour of 'regions'.")
+	}
+
+	if len(s.Regions) == 0 {
+		s.Regions = aliyunRegionList
+		s.Log.Warnf("'regions' is not set. Metrics will be queried across %d regions:\n%s",
```

Comment:
I think this is an `info` rather than a warning as it is a valid choice of the user isn't it?

Reply:
@i-prudnikov either I'm seeing some cached version of the file or you did not push your changes. For me it still says `Warnf`!?

Reply:
Forgot to push changes, now should be fixed